edgebuilder-install.sh

```diff
@@ -511,14 +516,20 @@ if [ "$COMPONENT" = "server" ];then
     exit 1
   fi
 elif [ "$COMPONENT" = "node" ]; then
-
-  if [ "$ARCH" = "x86_64" ]||[ "$ARCH" = "aarch64" ];then
+  if [ "$ARCH" = "x86_64" ];then
     if [ "$OS" = "$UBUNTU2204" ]||[ "$OS" = "$UBUNTU2004" ]||[ "$OS" = "$UBUNTU1804" ]||[ "$OS" = "$DEBIAN11" ]||[ "$OS" = "$DEBIAN10" ];then
       install_node "$OS" "$ARCH"
     else
       echo "ERROR: The Edge Builder node components are not supported on $OS - $ARCH"
       exit 1
     fi
+  elif [ "$ARCH" = "aarch64" ];then
```

Comment:
Jake, one more qn: could the machine/CPU arch be "arm64" as well?

Reply:
I haven't seen it in my testing. I think arm64 is just shorthand for aarch64

Reply:
Actually, aarch64 is the formal name for it apparently